zulip_bots/zulip_bots/bots/merels/test/test_database.py

```diff
@@ -3,64 +3,68 @@
 from libraries import database
 from libraries import game_data
 from zulip_bots.simple_lib import SimpleStorage
-
-
-class DatabaseTest(unittest.TestCase):
+from zulip_bots.test_lib import BotTestCase, DefaultTests
+from zulip_bots.game_handler import GameInstance
+from merels import MerelsHandler
+
+#Original: class DatabaseTest(unittest.TestCase):
+class DatabaseTest(BotTestCase, DefaultTests):
+    bot_name = 'merels'
+    
```

Comment:
`pep8      | zulip_bots/zulip_bots/bots/merels/test/test_database.py:13:1: W293 blank line contains whitespace` 
This error means there is "redundant white spaces" which need to be removed.
Just wondering what editor you're using as in VS code you can do something like this https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut.

Anyway, just observe the line number in the lint error and remove those whitespaces for othercases, e.g. in above case you can observe `13: 1`.

Reply:
Also, I am using Xcode

Reply:
I tried squashing the two committs and somehow it got many more commits that aren't even mine.  I am so confused because one the commit that had nothing on it did no even show up in the history so I don't know what I am actually squashing.  @shubhamdhama @showell

Reply:
ahh.. I think you did a merge or something, I'm not sure, so you should try rebasing or use `git reflog`,  the best tool when you think something went wrong. https://www.atlassian.com/git/tutorials/rewriting-history/git-reflog and `git reflog help` This may help you.
And feel free to open a thread at #devlopement help at chat.zulip.org, someone faced the similar condition can help you.

Reply:
@shubhamdhama Thank you! I have fixed the whitespace errors.  Should I commit my changes then squash the commits?

Reply:
There is just one commit so I think number of commit wise it's good(though I'm not sure the current commit title is the best to describe all the changes, it's upon steve what he thinks about title but still you should checkout our commit message guideline https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages)

And still there are lint errors, I would suggest first to fix the errors and test them locally and if everything goes right just amend the last commit.

Feel free to reach us at chat.zulip.org for any further discussion or doubt.